controllers/front/ProductController.php

```diff
@@ -806,14 +806,16 @@ public function getBreadcrumbLinks()
     {
         $breadcrumb = parent::getBreadcrumbLinks();
 
-        $categoryDefault = new Category($this->product->id_category_default);
+        $categoryDefault = new Category($this->product->id_category_default, $this->context->language->id);
```

Comment:
Just did it here: https://github.com/PrestaShop/PrestaShop/pull/5814/commits/5236a130becbae25a0e2c96982ea3babf52eaaee